src/openrct2/localisation/Date.cpp

```diff
@@ -65,6 +66,67 @@ void date_reset()
     gDateMonthsElapsed = 0;
     gDateMonthTicks = 0;
     gCurrentTicks = 0;
+    window_invalidate_by_class(WC_BOTTOM_TOOLBAR);
+}
+
+void date_set(sint32 year, sint32 month, sint32 day)
+{
+    year = clamp(1, year, 8192);
```

Comment:
With `date.c` renamed to `Date.cpp`, and now being compiled as C++, these should be changed to read `Math::Clamp` rather than `clamp`. Sorry about that.

Reply:
Changed